tests/unit/cryptoManager.spec.ts

```diff
@@ -8,15 +7,13 @@ import { ICryptoConfig } from '../../src/common/interfaces';
 
 // fsp module stubs
 let concatStub: jest.SpyInstance;
-let readFileStub: jest.SpyInstance;
 // crypto module stubs
 let createHashStub: jest.SpyInstance;
 let createCipherivStub: jest.SpyInstance;
 
 let cryptoManager: CryptoManager;
 
 const mockFileToSign = 'tests/mocks/files/mockTile.png';
-const mockKeyPem = '!%F=-?Pst970ss33445adfcF#-c3dafd';
```

Comment:
return this and use it to mock readFileSync before calling the constructor of the crypto manager